test/cartesian/CartesianGrid.spec.tsx

```diff
@@ -30,6 +30,179 @@ describe('<CartesianGrid />', () => {
     brushBottom: 7,
   };
 
+  const chartMargin: Margin = {
+    bottom: 11,
+    left: 12,
+    right: 13,
+    top: 14,
+  };
+
+  const expectedOffset: ChartOffset = {
+    bottom: 11,
+    brushBottom: 11,
+    height: 175,
+    left: 12,
+    right: 13,
+    top: 14,
```

Comment:
What would happen if these numbers were not to align with the chartMargin? If the alignment is required, why not just reuse it directly? 
```suggestion
   ...chartMargin
```

Reply:
I want this to be a little bit explicit, and I want the tests to fail when someone updates the margin but does not update the offset, and vice versa. If we DRY then people might miss the relationship between margin and offset here.